recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -83,40 +86,40 @@ def requirements(self):
         self.requires("gemmlowp/cci.20210928")
         if self.settings.arch in ("x86", "x86_64"):
             self.requires("intel-neon2sse/cci.20210225")
-        self.requires("ruy/cci.20210622")
+        self.requires("ruy/cci.20220628")
         if self.options.with_xnnpack:
-            self.requires("xnnpack/cci.20211210")
+            self.requires("xnnpack/cci.20220621")
+            self.requires("pthreadpool/cci.20210218")
```

Comment:
there's no specific call out for this and it's indirect https://conan.io/center/xnnpack?tab=dependencies

```suggestion
```